common/buildcraft/builders/TileQuarry.java

```diff
@@ -205,7 +205,7 @@ public void updateEntity() {
 			// In this case, since idling() does it anyway, we should return.
 			return;
 		} else if (stage == Stage.MOVING) {
-			int energyUsed = this.getBattery().useEnergy(20, (int) Math.ceil(20 + getBattery().getEnergyStored() / 10), false);
+			int energyUsed = this.getBattery().useEnergy(20, (int) Math.ceil(20D + (double) getBattery().getEnergyStored() / 10), false);
```

Comment:
> @psxlover - if either of the values is a double it does a double division. The Java compiler isn't that stupi.

Yeah, I wasn't talking about that, I just wasn't sure whether the casting applied before the division or after.